case/src/main/kotlin/com/ritense/case/deployment/CaseTabChangeset.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2015-2023 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.case.deployment
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.ritense.case.web.rest.dto.CaseTabDto
+
+class CaseTabChangeset(
+    val changesetId: String,
+    @JsonProperty("case-definitions")
+    val caseDefinitions: List<CaseDefinitionsTabCollection>
+)
+
+class CaseDefinitionsTabCollection(
```

Comment:
I think we should define public classes in their own file

Reply:
This could be package private i think. The only use of these classes is to represent the structure of the import file. I think having extra class files would only make the project structure less clear.